crates/astria-core/src/sequencer/v1alpha1/asset.rs

```diff
@@ -157,6 +157,12 @@ impl From<String> for Id {
     }
 }
 
+impl From<[u8; 32]> for Id {
```

Comment:
Is this used anywhere?

Reply:
yes, in `astria-sequencer/src/bridge/state_ext.rs:103`: `asset_ids.0.into_iter().map(asset::Id::from).collect()`